CDCgov/prime-reportstream
# Track how long the queries take to execute for the STLT dashboards

This PR tracks how long each query takes to execute for the delivery stlt dashboard.  Trying to track down why the queries execute relatively quickly in metabase but the API takes many magnitudes longer to finish

<img width="1347" alt="Screenshot 2023-08-29 at 4 44 57 PM" src="https://github.com/CDCgov/prime-reportstream/assets/1225895/97b68090-85f6-4574-a626-3ca63e7537b5">

Test Steps:
Run
```
curl --location 'http://localhost:7071/api/v1/receivers/ignore.HL7/deliveries' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer dummy' \
--data '{
    "sort": {
        "direction": "ASC",
        "property": "number_of_tests"
    },
    "pagination": {
        "page": 1,
        "limit": 10
    },
    "filters": [

    ]
}'
```
and verify you see the logging in the screenshot above.

## Changes
- Track how long the count, filtered count, results take to execute
- Track how long the delivery API takes to execute

## Checklist

### Testing

### Process

## Linked Issues


## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
